spacepy/pybats/bats.py

```diff
@@ -2601,51 +2736,55 @@ def calc_radflux(self, var, conv=1000. * (100.0)**3):
         Resulting value stored as "var_rflx".
         '''
 
-        if var+'_rflx' in self: return
-        
+        if var+'_rflx' in self:
+            return
+
         # Make sure we have radial velocity.
-        if 'ur' not in self: self.calc_urad()
+        if 'ur' not in self:
+            self.calc_urad()
 
         # Calc flux:
         self[var+'_rflx'] = self[var] * self['ur'] * conv
 
     @calc_wrapper
     def calc_radflu(self, var):
         '''
-        For variable *var*, calculate the radial fluence, or the 
-        spatially integrated radial flux through 2D surfaces of 
+        For variable *var*, calculate the radial fluence, or the
+        spatially integrated radial flux through 2D surfaces of
         constant radius.
 
         Resulting variable stored as "var_rflu".  Result will be an array
         with one value for each radial distance within the object.
         '''
 
         # Need at least 2D in angle space:
-        if self.dphi==0 or self.dtheta==0:
+        if self.dphi == 0 or self.dtheta == 0:
             raise ValueError('Fluence can only be calculated for 2D+ surfaces.')
-        
+
         # Trim flux off of val name:
-        if '_rflx' in var: var = var[:-5]
-        
+        if '_rflx' in var:
+            var = var[:-5]
+
         # Convenience:
         flux = var + '_rflx'
-        flu  = var + '_rflu'
-        
+        flu = var + '_rflu'
+
         # Make sure flux exists:
-        if flux not in self: self.calc_radflux(var)
-        if flu in self: return
+        if flux not in self:
+            self.calc_radflux(var)
+        if flu in self:
+            return
 
         # Create output container, one point per radial distance:
-        self[flu] = np.zeros( self['grid'][0] ) 
+        self[flu] = np.zeros(self['grid'][0])
 
         # Integrate over all radii.
         # Units: convert R to km and cm-3 to km.
         for i, R in enumerate(self['r']):
             self[flu] = np.sum(
-                (R* 6371.0)**2 * self[flux][i,:,:] * \
-                np.sin(self.theta[i,:,:])  * \
-                self.dtheta * self.dphi * 1000.**2 )
-
+                (R * 6371.0)**2 * self[flux][i, :, :] *
```

Comment:
Here it's 6371km (so could be `RE/1000`) but begs the question of consistency in Earth radius definition, as to what extent it matters here.